.github/workflows/charm-release.yaml

```diff
@@ -35,6 +35,13 @@ on:
           Can be used to turn off destructive mode while building.
           This is useful when building on other architectures/bases than the one
           of the runner.
+      build-preconfiguration:
```

Comment:
From what I'm seeing, the [charm-userdir-ldap](https://github.com/canonical/charm-userdir-ldap) needed it, however I think that that's just pure design and logic of `make bundled-packages-update`  should be moved to the charmcraft.yaml.